Failing to attach in OpenJDK 11

Using `ElasticApmAttacher.attach(Map<String, String> configuration)` on JDK11 logs the following error.

```stacktrace
Picked up JAVA_TOOL_OPTIONS: -XX:+UseG1GC -Dlogback.configurationFile=./logback.xml
OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended
Failed to start agent
java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at co.elastic.apm.agent.bci.AgentMain.init(AgentMain.java:85)
        at co.elastic.apm.agent.bci.AgentMain.agentmain(AgentMain.java:65)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:513)
        at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallAgentmain(InstrumentationImpl.java:535)
Caused by: java.lang.IllegalArgumentException: loader can't be null
        at co.elastic.apm.agent.shaded.dslplatform.json.DslJson$Settings.includeServiceLoader(DslJson.java:317)
        at co.elastic.apm.agent.shaded.dslplatform.json.DslJson$Settings.includeServiceLoader(DslJson.java:302)
        at co.elastic.apm.agent.shaded.dslplatform.json.DslJson.<init>(DslJson.java:424)
        at co.elastic.apm.agent.report.ApmServerHealthChecker.<init>(ApmServerHealthChecker.java:49)
        at co.elastic.apm.agent.report.ApmServerClient.<init>(ApmServerClient.java:82)
        at co.elastic.apm.agent.report.ApmServerClient.<init>(ApmServerClient.java:77)
        at co.elastic.apm.agent.impl.ElasticApmTracerBuilder.build(ElasticApmTracerBuilder.java:108)
        at co.elastic.apm.agent.bci.ElasticApmAgent.initialize(ElasticApmAgent.java:98)
        ... 12 more
```

Java: openjdk-11-jdk-headless:11.0.4+11-1ubuntu2~18.04.3
APM agent: co.elastic.apm:apm-agent-api:1.8.0
JAVA_HOME: /usr/lib/jvm/java-11-openjdk-amd64

The program is started via `java -jar`, with a `Class-Path:` entry in the manifest listing additional jars. The same thing happens with a Spring Boot jar.

--- a/apm-agent-core/src/main/java/co/elastic/apm/agent/report/ApmServerHealthChecker.java
+++ b/apm-agent-core/src/main/java/co/elastic/apm/agent/report/ApmServerHealthChecker.java
@@ -48,7 +48,7 @@ public class ApmServerHealthChecker implements Callable<Version> {
     private static final Logger logger = LoggerFactory.getLogger(ApmServerHealthChecker.class);
 
     private final ApmServerClient apmServerClient;
-    private final DslJson<Object> dslJson = new DslJson<>();
+    private final DslJson<Object> dslJson = new DslJson<>(new DslJson.Settings<>());
 
     public ApmServerHealthChecker(ApmServerClient apmServerClient) {
         this.apmServerClient = apmServerClient;

